contrib/initramfs/scripts/zfs

```diff
@@ -572,7 +572,7 @@ rollback_snap()
 # to the user to choose from.
 ask_user_snap()
 {
-	fs="$1"
+	ask_snap="$1"
```

Comment:
Technically not part of the original problem, but if we're going to rename parameter variables, better do it everywhere just to be safe and be consistent.

The parameter isn't a file system, it's a snapshot so rename it just to be consistent.